tests/components/nina/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,25 @@
+# serializer version: 1
+# name: test_diagnostics
+  dict({
+    'data': dict({
+      '083350000000': list([
+        dict({
+          '__type': "<class 'homeassistant.components.nina.coordinator.NinaWarningData'>",
+          'repr': "NinaWarningData(id='mow.DE-NW-BN-SE030-20201014-30-000', headline='Ausfall Notruf 112', description='Es treten Sturmböen mit Geschwindigkeiten zwischen 70 km/h (20m/s, 38kn, Bft 8) und 85 km/h (24m/s, 47kn, Bft 9) aus westlicher Richtung auf. In Schauernähe sowie in exponierten Lagen muss mit schweren Sturmböen bis 90 km/h (25m/s, 48kn, Bft 10) gerechnet werden.', sender='Deutscher Wetterdienst', severity='Minor', recommended_actions='', affected_areas='Gemeinde Oberreichenbach, Gemeinde Neuweiler, Stadt Nagold, Stadt Neubulach, Gemeinde Schömberg, Gemeinde Simmersfeld, Gemeinde Simmozheim, Gemeinde Rohrdorf, Gemeinde Ostelsheim, Gemeinde Ebhausen, Gemeinde Egenhausen, Gemeinde Dobel, Stadt Bad Liebenzell, Stadt Solingen, Stadt Haiterbach, Stadt Bad Herrenalb, Gemeinde Höfen an der Enz, Gemeinde Gechingen, Gemeinde Enzklösterle, Gemeinde Gutach (Schwarzwaldbahn) und 3392 weitere.', web='https://www.wettergefahren.de', sent='2021-10-11T05:20:00+01:00', start='2021-11-01T05:20:00+01:00', expires='3021-11-22T05:19:00+01:00', is_valid=True)",
```

Comment:
This looks hard to read. Use dataclass asdict on the data before returning it as diagnostics.